cisco_secure_firewall/assets/logs/cisco-secure-firewall_tests.yaml

```diff
@@ -0,0 +1,142 @@
+id: cisco-secure-firewall
+tests:
+  - sample: "<190>2024-01-17T09:07:34Z: %FTD-2-109011: Authen Session Start: user
+      testuser1, sid 23"
+    result:
+      custom:
+        message_id: "109011"
+        severity: 2
+        syslog:
+          severity: "Critical"
```

Comment:
Is this expected? The sample seems like a regular log. If this shouldn't be marked as critical, some updates to the pipeline would be required

It appears like this within the app: 
![image](https://github.com/DataDog/integrations-core/assets/124740576/37e40d8d-8c8d-4a23-be7b-98693474c88c)

Reply:
Based on the Cisco Secure Firewall documentation they have described the severity levels as mentioned below.
![image](https://github.com/DataDog/integrations-core/assets/105789910/0dbd94c2-be96-4911-909b-32005736576d)
Also, here is the documentation link that describe the syslog message format for cisco secure firewall. Which says the level is the severity of the syslog message hence we have mapped it to syslog.severity: https://www.cisco.com/c/en/us/td/docs/security/firepower/Syslogs/b_fptd_syslog_guide/about.html#:~:text=Event%20Syslog%20Alerting-,Syslog%20Message%20Format,-Syslog%20messages%20are